doc/ntp/ntp-design.md

```diff
@@ -189,6 +193,8 @@ Addresses are classed by type as (s) a remote server or peer (IPv4 class A, B an
 | key (ntpd) | none    | All packets sent to and received from the server or peer are to include authentication fields encrypted using the specified key identifier with values from 1 to 65535, inclusive. The default is to include no encryption field. |
 | version    | 4       | Specifies the version number to be used for outgoing NTP packets. Versions 1-4 are the choices, with version 4 the default. |
 | iburst     | none    | When the server is unreachable, send a burst of eight packets instead of the usual one. The packet spacing is normally 2 s; however, the spacing between the first two packets can be changed with the calldelay command to allow additional time for a modem or ISDN call to complete. This is designed to speed the initial synchronization acquisition with the server command and s addresses and when ntpd is started with the -q option. |
+| minpoll    | 6        | |
```

Comment:
Add some description for minpoll, even it is similar as maxpoll

Reply:
done.